tests/keras/engine/test_training.py

```diff
@@ -619,17 +803,11 @@ def test_model_with_external_loss():
 
     optimizer = 'rmsprop'
     loss = None
-    model.compile(optimizer, loss, metrics=['mae'])
+    model.compile(optimizer, loss, metrics=['mse'])
```

Comment:
I didn't realize there was mean average error. It doesn't affect the test outcome or purpose but I can put it back if needed.